config/eslintrc.json

```diff
@@ -47,33 +47,6 @@
     "worker": true,
     "mocha": true
   },
-  "globals": {
-    // particle implementations
-    "defineParticle": false,
```

Comment:
I tried pulling this to my local to see the impact. It seems to not cause any problems. 
I was hesitant to remove `defineParticle`, but that seems fine. 

LGTM.